### PR TITLE
Create release-notes.md

### PR DESCRIPTION
**Pull request by sandysage**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Create a Lockbox account with a master password to lock/unlock your data
+* View entries
+* Search entries
+* Add, edit, and delete entries
+* Copy username and password to clipboard
+* Submit feedback
+
+### Known Issues
+
+* There is no way to reset your Lockbox master password. If you forget your master password, you'll need to create a new profile and start fresh.
```
Comment (m8ttyB) [inline docs/release-notes.md]: This is a bit misleading; uninstalling and then re-installation resets the master password as well as the local datastore.

Comment (linuxwolf) [reply docs/release-notes.md]: @m8ttyB thanks.  I must have really screwed up my dev profile -- I was having issues with this, but didn't explore further.

**Comment by linuxwolf** [comment]
@jimporter any objections to merging this?

**Comment by linuxwolf** [comment]
merging without a review: latest push was only to rebase to `master`.

**Pull request merged**